### PR TITLE
impl Book, BookRepository, BookService

### PR DESCRIPTION
**Pull request by Petro-Smoliar**

### CODE REVIEW

File: src/main/java/com/example/book/store/config/AppConfig.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import javax.sql.DataSource;
+import org.apache.commons.dbcp.BasicDataSource;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.orm.hibernate5.LocalSessionFactoryBean;
+
+@Configuration
+@ComponentScan(basePackages = "com.example.book.store")
+public class AppConfig {
```
Comment (Ivan95kos) [inline src/main/java/com/example/book/store/config/AppConfig.java]: In the first steps, it is better to use the default Beans
You can remove this config

File: src/main/java/com/example/book/store/model/Book.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+
+@Entity
+@Table(name = "books")
+public class Book {
```
Comment (Ivan95kos) [inline src/main/java/com/example/book/store/model/Book.java:16]: Use the library Lombok
then you will be able to use the annotations like @Data, @Setter, @Getter

File: src/test/java/com/example/book/store/BookStoreApplicationTests.java [hunk 3]
```diff
@@ -1,13 +1,9 @@
-package com.example.book_store;
+package com.example.book.store;
 
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
 
 @SpringBootTest
```
Comment (Ivan95kos) [inline src/test/java/com/example/book/store/BookStoreApplicationTests.java]:
```suggestion
@SpringBootTest(classes = BookStoreApplicationTests.class)
```

File: src/test/java/com/example/book/store/BookStoreApplicationTests.java [hunk 4]
```diff
@@ -1,13 +1,9 @@
...
 class BookStoreApplicationTests {
 
-	@Test
-	void contextLoads() {
-	}
```
Comment (Ivan95kos) [inline src/test/java/com/example/book/store/BookStoreApplicationTests.java:11]: This test must be present, revert it

**Review by Ivan95kos — CHANGES_REQUESTED** [review]
Good job overall!

**Review by kozhukhovsky — APPROVED** [review]

**Review by kozhukhovsky — APPROVED** [review]

**Pull request merged**